.gitlab-ci.yml

```diff
@@ -71,15 +71,34 @@ check-line-width:
   allow_failure:                   false
 
 
+cargo-audit:
```

Comment:
Shouldn't this be run on a daily basis for the master + send the results to somewhere. Maybe into a dedicated Riot channel?

Reply:
Failures are pretty rare here (for now). It will be very spammy to send notifications about it every time it fails.
Everyone will see these "allowed to fail" exlamation marks in gitlab CI and red crosses here as well.

Reply:
The problem is, that people are already confused about the `check-runtime` pipeline, especially if that fails. 
To prevent spamming, I proposed to let it run once per day :) 
I think we should disable it, until it is an independently running job :)

Reply:
maybe it could be run on merges to master only. imo it's more a question of who should react upon failures. for instance one could think of a nightly run which opens an issue on github if something needs attention.

Reply:
Let's try it this way for a while. Then change if needed.